### PR TITLE
Display and check if julia & stdlib versions match

### PR DESCRIPTION
**Pull request by IanButterworth**
Trying to be helpful in keeping the versions of stdlibs and base julia in-step

### CODE REVIEW

File: src/bump-stdlibs.jl [hunk 1]
```diff
@@ -130,12 +131,16 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                         git_url_markdown = _git_url_to_formatted_markdown(stdlib.git_url)
                         bumpstdlibs_sender = strip(get(ENV, "BUMPSTDLIBS_SENDER", ""))
                         bumpstdlibs_sender_ping = isempty(bumpstdlibs_sender) ? "unknown user" : "@$(bumpstdlibs_sender)"
+                        julia_version = Base.VersionNumber(Base.parsed_toml("Project.toml")["version"])
```
Comment (DilumAluthge) [inline src/bump-stdlibs.jl:134]: Is this a typo? Presumably you should open the `VERSION` file from the Julia source tree, and use that?

File: src/bump-stdlibs.jl [hunk 2]
```diff
@@ -107,6 +107,7 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                 stdlib_current_commit_in_upstream_short = strip(read(`git rev-parse --short $(stdlib_current_commit_in_upstream)`, String))
                 assert_string_startswith(stdlib_current_commit_in_upstream, stdlib_current_commit_in_upstream_short)
                 run(`git fetch --all --prune`)
+                stdlib_version = Base.VersionNumber(Base.parsed_toml("Project.toml")["version"])
```
Comment (DilumAluthge) [inline src/bump-stdlibs.jl:110]: FWIW, I'd be fine with adding TOML as a dependency of this package, and then you could use TOML (instead to needing to use `Base.parsed_toml`).

**Comment by DilumAluthge** [comment]
Hmmm why isn't CI running?

**Comment by DilumAluthge** [comment]
I guess maybe a concurrency group somewhere is waiting to be freed up.

File: src/bump-stdlibs.jl (hunk 2 above)
Comment (IanButterworth) [reply src/bump-stdlibs.jl]: What's wrong with `Base.parsed_toml`?

**Comment by IanButterworth** [comment]
There's a global lock on integration tests. Old actions need to be stopped first. Feel free to add me as a collaborator to do that

**Comment by IanButterworth** [comment]
weird.. is the checksum for Statistics wrong on julia master?

**Review by DilumAluthge — APPROVED** [review]

File: src/bump-stdlibs.jl [hunk 3]
```diff
@@ -107,6 +107,15 @@ function _bump_single_stdlib(stdlib::StdlibInfo, config::Config, state::State)
                 stdlib_current_commit_in_upstream_short = strip(read(`git rev-parse --short $(stdlib_current_commit_in_upstream)`, String))
                 assert_string_startswith(stdlib_current_commit_in_upstream, stdlib_current_commit_in_upstream_short)
                 run(`git fetch --all --prune`)
+                stdlib_version = if isfile("Project.toml")
+                    try
+                        Base.VersionNumber(TOML.parse("Project.toml")["version"])
```
Comment (DilumAluthge) [inline src/bump-stdlibs.jl:112]: Should this be `TOML.parsefile`?

**Review by DilumAluthge — APPROVED** [review]

**Pull request merged**